taskflow_test.go

```diff
@@ -210,9 +204,7 @@ func Test_task_panics(t *testing.T) {
 func Test_cancelation(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	cancel()
-	flow := &taskflow.Taskflow{
-		Output: ioutil.Discard,
-	}
+	flow := &taskflow.Taskflow{}
```

Comment:
why not `taskflow.New()`? :P

Reply:
I want to make sure that the struct is useful even when not all fields are initialized. However I will consider to use `New` in some tests

Reply:
Made some changes here: https://github.com/pellared/taskflow/pull/11/commits/dd5584a5b0fc0cf06babdb7f5f9059e5ba08e16f